### DISCUSSION TITLE
Device Unique ID (duid) on Netbox

### DISCUSSION THREAD

**Discussion by jscmenezes** [netbox-community/netbox#6550] [Q&A] [answered]
Hi,

How (or where) can I register the Device Unique ID for DHCPv6 purposes?

Thanks in advance!

Jeronimo

**Comment by candlerb** [comment, marked as answer]
I'd suggest a [Custom Field](https://netbox.readthedocs.io/en/stable/additional-features/custom-fields/) on Device and VM.  As I understand it, the [same DUID is used on all interfaces](https://datatracker.ietf.org/doc/html/rfc8415#section-11.2).

If you wanted to make it a Custom Field on Interface, then that's also available since Netbox v2.11.
